frontend/public/components/utils/dropdown.jsx

```diff
@@ -319,6 +319,7 @@ export class Dropdown extends DropdownMixin {
     };
 
     _.each(items, (v, k) => addItem(k, v));
+    const sortedRows = _.sortBy(rows, ['props', 'content']);
```

Comment:
We should *not* be changing the order of the items passed in. This will affect every dropdown and override the order set by the outer component.

We should also avoid sorting in render if possible. Some of these dropdowns like the projects dropdown can be big (thousands of items).

Reply:
Isn't `content` also potentially a JSX element? I'm not sure how sort would work in that case.

Reply:
HI @spadgett, should we strive to restore the item to it's original position in the dropdown after it is unbookmarked?

Reply:
That's probably how it should work.

I'd suggest deferring this until after 4.0. The dropdown component is complicated and used in many places. We have several bugs around how it handles sorting, but reworking that at this point in the release is risky.

The current proposed fix likely exacerbates https://jira.coreos.com/browse/CONSOLE-770 for instance. We also have problems with projects whose name begins with a number because we rely on key insertion order. But fixing these problems is not simple.